dialect/sql/schema/mysql.go

```diff
@@ -280,6 +280,9 @@ func (d *MySQL) cType(c *Column) (t string) {
 		t = fmt.Sprintf("enum(%s)", strings.Join(values, ", "))
 	case field.TypeUUID:
 		t = "char(36) binary"
+		if d.supportsUUID() {
```

Comment:
Let me check first if a migration from previous type to the new uuid works seamlessly or we need to do something.

Reply:
Any update on this one?, 

I just rebased this PR with main and tested if it still works as expected. My other changes inside atlas where merged and released in v0.9 3 week ago. 

It would be great if we can merge this PR asap.